landing-page/components/sections/tasks/FeatureSection.tsx

```diff
@@ -1,8 +1,40 @@
 import { Span } from '@helpwave/common/components/Span'
 import { tw, tx } from '@helpwave/common/twind'
 import Image from 'next/image'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsForTranslation } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
 
-const FeatureSection = () => {
+type FeatureSectionTranslation = {
+  taskTemplatesTitle: string,
+  taskTemplatesText: string,
+  collaborateTitle: string,
+  collaborateText: string,
+  patientsTitle: string,
+  patientsText: string
+}
+
+const defaultFeatureSectionTranslation: Record<Languages, FeatureSectionTranslation> = {
+  en: {
+    taskTemplatesTitle: 'Task Templates',
+    taskTemplatesText: 'Save repetitive tasks as templates. Standardize your workflows, share them with your team.',
+    collaborateTitle: 'Collaborate',
+    collaborateText: 'You and your team synchronized on the same heartbeat.',
+    patientsTitle: 'Patients',
+    patientsText: 'Modern dashboards for your patients. Keep track of their progress and tasks.'
+  },
+  de: {
+    taskTemplatesTitle: 'Aufgabenvorlagen',
+    taskTemplatesText: 'Speichere sich wiederholende Aufgaben als Vorlagen. Standardisiere deine Arbeitsabläufe und teile sie mit deinem Team.',
+    collaborateTitle: 'Zusammenarbeiten',
+    collaborateText: 'Du und dein Team arbeiten auf den Herzschlag um Herzschlag synchronisiert zusammen.',
```

Comment:
Maybe we should not translate it literally
```suggestion
    collaborateText: 'Du und dein Team arbeiten auf den Herzschlag synchronisiert zusammen.',
```